### ISSUE TITLE
nodai_shark_studio_20231009_984.exe does not work

### ISSUE THREAD

**Issue by kivanctun** [nod-ai/SHARK#1908]
when I try to run the nodai_shark_studio_20231009_984.exe  , I get following error. 

G:\nodai_shark_studio>nodai_shark_studio_20231009_984.exe
[26036] Failed to extract PIL\_imaging.cp311-win_amd64.pyd: decompression resulted in return code -1!

G:\nodai_shark_studio>

**Comment by Acrivec** [comment]
Redownload imho, looks like damaged file

**Comment by monorimet** [comment]
Yes, that specific release is broken. Thanks for the heads up, I'll mark it as such.
